pkg/containerwatcher/v1/container_watcher.go

```diff
@@ -224,9 +224,17 @@ func CreateIGContainerWatcher(cfg config.Config, applicationProfileManager appli
 		if event.K8s.ContainerName == "" {
 			return
 		}
+
+		// ignore DNS events that are not responses
 		if event.Qr != tracerdnstype.DNSPktTypeResponse {
 			return
 		}
```

Comment:
Is this new? Or we got ".internal" domain request together with this?

Reply:
This is not new, we get internal also with response events:
```
% kubectl gadget trace dns -n kubescape
NODE                                                                NAMESPACE                                                           POD                                                                 PID        TID        COMM             QR TYPE      QTYPE      NAME                                                                RCODE                               NUMANSW…
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         Q  OUTGOING  AAAA       otel-collector.kubescape.svc.cluster.local.                                                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         Q  OUTGOING  A          otel-collector.kubescape.svc.cluster.local.                                                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         R  HOST      A          otel-collector.kubescape.svc.cluster.local.                         NoError                             1
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         R  HOST      AAAA       otel-collector.kubescape.svc.cluster.local.                         NoError                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         R  HOST      A          otel-collector.kubescape.svc.cluster.local.                         NoError                             1
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         R  HOST      AAAA       otel-collector.kubescape.svc.cluster.local.                         NoError                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         Q  OUTGOING  AAAA       otel-collector.kubescape.svc.cluster.local.                                                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         Q  OUTGOING  A          otel-collector.kubescape.svc.cluster.local.                                                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         Q  OUTGOING  A          gateway.kubescape.svc.cluster.local.                                                                    0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         Q  OUTGOING  AAAA       gateway.kubescape.svc.cluster.local.                                                                    0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         R  HOST      AAAA       gateway.kubescape.svc.cluster.local.                                NoError                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         R  HOST      A          gateway.kubescape.svc.cluster.local.                                NoError                             1
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         Q  OUTGOING  AAAA       kubescape.kubescape.svc.cluster.local.                                                                  0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         Q  OUTGOING  AAAA       kubevuln.kubescape.svc.cluster.local.                                                                   0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         Q  OUTGOING  A          kubescape.kubescape.svc.cluster.local.                                                                  0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         Q  OUTGOING  A          kubevuln.kubescape.svc.cluster.local.                                                                   0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         R  HOST      AAAA       kubevuln.kubescape.svc.cluster.local.                               NoError                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         R  HOST      AAAA       kubescape.kubescape.svc.cluster.local.                              NoError                             0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         R  HOST      A          kubevuln.kubescape.svc.cluster.local.                               NoError                             1
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         R  HOST      A          kubescape.kubescape.svc.cluster.local.                              NoError                             1
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         Q  OUTGOING  A          report.armo.cloud.kubescape.svc.cluster.local.                                                          0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         Q  OUTGOING  AAAA       report.armo.cloud.kubescape.svc.cluster.local.                                                          0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860640     operator         R  HOST      A          report.armo.cloud.kubescape.svc.cluster.local.                      NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860623     operator         R  HOST      AAAA       report.armo.cloud.kubescape.svc.cluster.local.                      NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         Q  OUTGOING  AAAA       report.armo.cloud.svc.cluster.local.                                                                    0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         Q  OUTGOING  A          report.armo.cloud.svc.cluster.local.                                                                    0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         R  HOST      A          report.armo.cloud.svc.cluster.local.                                NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         R  HOST      AAAA       report.armo.cloud.svc.cluster.local.                                NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         Q  OUTGOING  A          report.armo.cloud.cluster.local.                                                                        0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         Q  OUTGOING  AAAA       report.armo.cloud.cluster.local.                                                                        0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         R  HOST      A          report.armo.cloud.cluster.local.                                    NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860641     operator         R  HOST      AAAA       report.armo.cloud.cluster.local.                                    NXDomain                            0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860637     operator         Q  OUTGOING  AAAA       report.armo.cloud.us-central1-c.c.armo-test-clusters.internal.                                          0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         Q  OUTGOING  A          report.armo.cloud.us-central1-c.c.armo-test-clusters.internal.                                          0
gke-dwertent-syft-default-pool-d4e9ae18-b0tw                        kubescape                                                           operator-64ff968c5c-gmgfv                                           860623     860638     operator         R  HOST      A          report.armo.cloud.us-central1-c.c.armo-test-clusters.internal.      NXDomain                            0
```